bin/bridge-client/src/modes/operator/bootstrap.rs

```diff
@@ -158,8 +158,11 @@ pub(crate) async fn bootstrap(args: Cli) -> anyhow::Result<()> {
         msg_polling_interval,
     };
 
+    let shared_exec_handler = Arc::new(exec_handler);
+    let exec_handler_clone = shared_exec_handler.clone();
+
     let task_manager = TaskManager {
-        exec_handler: Arc::new(exec_handler),
+        exec_handler: exec_handler_clone,
```

Comment:
Can just do `shared_exec_handler.clone()`.

Conventionally, you don't need to add the `shared_` prefix either fwiw.